src/backend/src/transformers/auth-user.transformer.ts

```diff
@@ -11,7 +12,8 @@ const authenticatedUserTransformer = (user: Prisma.UserGetPayload<typeof authUse
     emailId: user.emailId,
     role: user.role,
     defaultTheme: user.userSettings?.defaultTheme,
-    teamAsLeadId: user.teamAsLead?.teamId
+    teamAsLeadId: user.teamAsLead?.teamId,
+    favoriteProjects: user.favoriteProjects.map((project) => projectTransformer(project))
```

Comment:
above comment, might make this just map the project ids, then make a get request in the frontend when we need it.